util/find-doc-nits.pl

```diff
@@ -173,10 +192,83 @@ ()
     unlink $temp || warn "Can't remove $temp, $!";
 }
 
-getopts('s');
+my %dups;
+
+sub parsenum()
+{
+    my $file = shift;
+    my @apis;
+
+    open my $IN, '<', $file
+        or die "Can't open $file, $!, stopped";
+
+    while ( <$IN> ) {
+        next if /\sNOEXIST/;
```

Comment:
Side note. `\b` is likely to reflect intention better. And having said that `\bNOEXIST\b` is probably appropriate. And having said that it might be appropriate to give same treatment to EXPORT_VAR_AS_FUNC. "Side note" means that reaction is considered non-obligatory.